src/Symfony/Component/HttpFoundation/IpUtils.php

```diff
@@ -120,8 +120,8 @@ public static function checkIp4(string $requestIp, string $ip): bool
     public static function checkIp6(string $requestIp, string $ip): bool
     {
         $cacheKey = $requestIp.'-'.$ip.'-v6';
-        if (isset(self::$checkedIps[$cacheKey])) {
-            return self::$checkedIps[$cacheKey];
+        if (null !== ($cacheValue = self::getCacheResult($cacheKey))) {
```

Comment:
extra brackets should be removed
but more importantly,this line need to by duplicated on L78 ALSO